datasets/br/br_tcu_disqualified/br_tcu_disqualified.yaml

```
@@ -0,0 +1,24 @@
+title: Brazil's TCU list of individuals disqualified from public service
+entry_point: crawler.py
+prefix: br-tcu-disqualified
```

```suggestion
prefix: br-tcu-disq
```

Let's try and go for shorter prefixes - they prefix all the generated IDs - in this case the sanctions